### DISCUSSION TITLE
I would like to know what the units in peacock are

### DISCUSSION THREAD

**Discussion by lei123abc** [idaholab/moose#25459] [Q&A Tools] [answered]

**Comment by dschwen** [comment, marked as answer]
There are no intrinsic units. They are whatever your input dictates and what you decide your parameter units are. If your diffusion coefficient is in m^2/s then your length units will be m and your time units will be s.

**Reply by lei123abc** [reply]
I got it,thank you for your answer.

**Reply by GiudGiud** [reply]
the main item that determines the units to use is usually the units that you used to make your mesh. 
Everything has to be consistent with your mesh

**Reply by lei123abc** [reply]
I'm a little confused now, does it depend on the units of my mesh or the units of the input file? For example, the mesh uses millimeter units, but the input file uses m. After the moose calculation is completed, is the displacement mm or m?

**Reply by GiudGiud** [reply]
they both need to be consistent. You cannot have the mesh in mm and the input in m. You can scale back your mesh to meters using the TransformGenerator btw.

**Reply by lei123abc** [reply]
OK,thank you.I'll try.
